### ISSUE TITLE
'neoscrypt_cpu_hash_k4_2sream' at line 1480 :the launch timed out and was terminated

### ISSUE THREAD

**Issue by Toxias** [KlausT/ccminer#164]
How to fix it:
'neoscrypt_cpu_hash_k4_2sream' at line 1480 :the launch timed out and was terminated

New mining rig with instaled only Windows 10 PRo 64bit And Asus Motherboard Drivers B250 Mining Expert Newest Bios, and newest Nvidia drivers. After startign mining proces i see this error.

**Comment by Toxias** [comment]
![image](https://user-images.githubusercontent.com/34247626/35184035-5162a3f8-fdf0-11e7-89b5-76ef63510ac2.png)

**Comment by Toxias** [comment]
![image](https://user-images.githubusercontent.com/34247626/35184082-1ce19fca-fdf1-11e7-904c-d1bb7ebef673.png)

**Comment by Toxias** [comment]
After instaling new nvidia drivers but dont restarting system all work fine: 390.65-desktop-win10-64bit-international-whql

![image](https://user-images.githubusercontent.com/34247626/35184239-3eb9cd0a-fdf3-11e7-9afc-bf2d430a15da.png)


But after restarting system erorrs come back again?!

**Comment by ultiman2** [comment]
i have the same error with version 8.19pre after some time ccminer crashed with a error message. i revert back to 8.18 and so for no more error of that kind

**Comment by KlausT** [comment]
When you get "the launch timed out" then there is something wrong with your system.
Do you overclock?

**Comment by ultiman2** [comment]
i overclock a little bit but like i said didnt do this error with version
8.18 so something changed in 8.19pre

**Comment by KlausT** [comment]
You can try this version if you want:
[ccminer-819-jan20-cuda91-x64.zip](https://github.com/KlausT/ccminer/files/1648953/ccminer-819-jan20-cuda91-x64.zip)

**Comment by kaos-1** [comment]
I had this same issue happen today using 8.1.8 windows 64 binaries , i couldn't wrap my head around it.
Restarting the rig, and changing the nicehash server to US from EU fixed it.
Its the first time i saw this error.

**Comment by KlausT** [comment]
Yeah, restarting is always a good idea when wierd things are happening

**Comment by ultiman2** [comment]
just trying this version, this version fix the crash on exit in windows?

**Comment by KlausT** [comment]
I don't know if it is fixed. It has never crashed on my system.

**Comment by Toxias** [comment]
My System is new Windows 10 with nvidia newest drivers i try with oc and without OC and its still crash

**Comment by Toxias** [comment]
I was using one of my mining cart to comunicate with monitor by display port cabel when i disconect it and i conect monitor by HDMI to my motherboard build in cpu graphic card i think all work fine no crash no errors? If somone other it will help please tell me this.

**Comment by rocketeee** [comment]
@KlausT hi there,recently, i was wondering why u r the only one that not crash. then i did some work on the issues people are talking abaut like this one 'neoscrypt_cpu_hash_k4_2sream' stuff.....and I found that there is some thing with hardware problems. i did swap and replace some of the raiser and the cable and some of my gpus, the crashing situations are not the same. finally i found the raiser is the part that most vulnerable, only 75% of the raisers are going well with neoscrypt algo while 100% ok with other algos like nist5. maybe the algo needs more power supply and the raiser is not capable

**Comment by rocketeee** [comment]
@KlausT I saw some other developers mentioned that they did some work on optimizing with the raisers, since im not an expert on crypto development i do not know how they do it,but may be you can try it. Thanks again for your great work with ccminer, its the fastest miner with no OC inside as i know

**Comment by lowenheart** [comment]
I have got that error but when I reduce the memory clock its going fine
